x/cronos/keeper/evm_log_handlers.go

```diff
@@ -30,11 +32,16 @@ var (
 	// EthereumTransferEvent represent the signature of
 	// `event __CronosSendToEthereum(address recipient, uint256 amount, uint256 bridge_fee)`
 	EthereumTransferEvent abi.Event
+
+	// IbcTransferEvent represent the signature of
+	// `event __CronosSendToIbc(address sender, string recipient, uint256 amount)`
```

Comment:
```suggestion
	// `event __CronosSendToIbc(string recipient, uint256 amount)`
```

It don't have the sender parameter in the definition.

Reply:
fixed